components/app.js

```diff
@@ -6,50 +6,47 @@ import Button from './button';
 import Clock from './clock';
 import Score from './score';
 
+const App = () => {
+  return (
+    <main>
+      <header>
+        <div className="container">
+          <h1>BOGGLE</h1>
+          <Button />
+          <Attempt />
+        </div>
+      </header>
 
-class App extends React.Component {
-  render() {
-    return (
-      <main>
-        <header>
-          <div className="container">
-            <h1>BOGGLE</h1>
-            <Button />
-            <Attempt />
-          </div>
-        </header>
+      <section>
+        <div className="container">
+          <Board />
+          <aside>
+            <Clock />
+            <Score />
+          </aside>
+        </div>
+      </section>
 
-        <section>
-          <div className="container">
-            <Board />
-            <aside>
-              <Clock />
-              <Score />
-            </aside>
+      <footer>
+        <div className="container">
+          <div className="copyright">
+            &copy; Copyright 2016 Leonardo Quixadá. All rights reserved.
+            <a href="https://travis-ci.org/lquixada/boggle">
+              <img src="https://travis-ci.org/lquixada/boggle.svg?branch=master" className="build-status" />
+            </a>
           </div>
-        </section>
-
-        <footer>
-          <div className="container">
-            <div className="copyright">
-              &copy; Copyright 2016 Leonardo Quixadá. All rights reserved.
-              <a href="https://travis-ci.org/lquixada/boggle">
-                <img src="https://travis-ci.org/lquixada/boggle.svg?branch=master" className="build-status" />
-              </a>
-            </div>
 
-            <div className="social">
-              <ul>
-                <li><a href="https://github.com/lquixada/"><img src="images/logo-github.png" /></a></li>
-                <li><a href="https://facebook.com/lquixada/"><img src="images/logo-facebook.png" /></a></li>
-                <li><a href="https://twitter.com/lquixada/"><img src="images/logo-twitter.png" /></a></li>
-              </ul>
-            </div>
+          <div className="social">
+            <ul>
+              <li><a href="https://github.com/lquixada/"><img src="images/logo-github.png" /></a></li>
+              <li><a href="https://facebook.com/lquixada/"><img src="images/logo-facebook.png" /></a></li>
+              <li><a href="https://twitter.com/lquixada/"><img src="images/logo-twitter.png" /></a></li>
+            </ul>
           </div>
-        </footer>
-      </main>
-    );
-  }
+        </div>
+      </footer>
+    </main>
+  )
 }
 
 export default App;
```

Comment:
this one I already knew. I left it like this intentionally just to keep a standard throughout the components. Since both ways lead to the same results, I tend to favor standards.